src/main/python/api.py

```diff
@@ -0,0 +1,33 @@
+from flask import Flask, jsonify, request
+import argparse
+
+app = Flask(__name__)
+
+@app.route("/", methods=['GET'])
+def hello():
+    return "Hello! The server is working properly... :)"
+
```

Comment:
This is the root route. I think it's ok to leave it here to make sure the server works. :)